svc/IdentityService.ts

```diff
@@ -159,4 +160,11 @@ export class IdentityService extends HoistService {
         }
         return false;
     }
+
+    private canUserImpersonate(user: HoistUser): boolean {
+        const hasPerm = checkMinVersion(XH.environmentService.get('hoistCoreVersion'), '16.3.0')
+            ? user.hasRole(`HOIST_IMPERSONATOR`)
+            : user.isHoistAdmin;
+        return hasPerm && XH.getConf('xhEnableImpersonation');
```

Comment:
I think it's the right call here to remove the default value from the getConf call - we've had that config auto-bootstrapped by hoist-core for a while now.